### PR TITLE
Task Track - Index

### PR DESCRIPTION
**Pull request by leonidbogin**
image

### CODE REVIEW

**Review by SashaDrapun — APPROVED** [review]
Good job

File: DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿using DIMS_Core.BusinessLayer.Models.TaskTrack;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace DIMS_Core.BusinessLayer.Interfaces
+{
+    public interface ITaskTrackService
+    {
+        Task<IEnumerable<VUserTrackModel>> GetAllForMember(int UserId);
```
Comment (PushMeNow) [inline DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs:9]: Camelcase for method params.

Comment (PushMeNow) [reply DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs:9]: Rename please method. If you are looking for by smth method name should be named respectively (example, GetAll**By**UserId).

**Review by PushMeNow — CHANGES_REQUESTED** [review]
Hi.
I did some comments. Look them through and fix. Let me know if any questions.

**Review by SashaDrapun — APPROVED** [review]
Good Job!

File: DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs (hunk 1 above)
Comment (leonidbogin) [reply DIMS-Core/DIMS-Core.BusinessLayer/Interfaces/ITaskTrackService.cs:9]:
```suggestion
        Task<IEnumerable<VUserTrackModel>> GetAllByUserId(int userId);

**Review by PushMeNow — APPROVED** [review]
Good job.

**Pull request merged**